superx/information_extractors/branch_info_extractor.py

```diff
@@ -1,17 +1,25 @@
+'''
+imports
+'''
 import gzip
-from bs4 import BeautifulSoup
+#pylint: disable=import-error
 import xml.etree.ElementTree as et
-from app import supermarket_info_dictionary, session, db
-from models import Branch
 import logging
 import requests
+from bs4 import BeautifulSoup
+from app import supermarket_info_dictionary, session
+from models import Branch
```

Comment:
this is a problematic. see my pr Extractor -automation. i changed the way the imports are and added a path to them between imports.

Reply:
its problematic because they wau i need the imports to be now wont pass the test.

Reply:
@aryehlev - please respond

Reply:
ok fixed to yours